spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb

```diff
@@ -211,6 +211,16 @@ def assert_specific_vm_powered_on_hardware_networks(v)
     )
   end
 
+  def assert_specific_vm_powered_disk(v)
```

Comment:
this method name reads funny (as does the method before it).  You aren't asserting the power state of the vm, so having that in the name of the method is incorrect.  Can this just be called `assert_specific_disk`?  Also, the point of the "specific" word in the method name is that it looks at a specific Vm, but this method takes a parameter...it just feels weird, basically, there is a parameter, but there is only one specific Vm that can go into the parameter.

For the previous method, can we just call it `assert_specific_hardware_networks`?